aiida_vasp/workchains/relax.py

```diff
@@ -313,9 +314,12 @@ def verify_next_workchain(self):
             self.ctx.exit_code = compose_exit_code(next_workchain_exit_status, next_workchain_exit_message)
             self.report('The called {}<{}> returned a non-zero exit status. '
                         'The exit status {} is inherited'.format(workchain.__class__.__name__, workchain.pk, self.ctx.exit_code))
-            # Make sure at the very minimum we attach the misc node that contains notifications and other
+            # Make sure at the very minimum we attach the misc node (if it exists) that contains notifications and other
             # quantities that can be salvaged
-            self.out('misc', workchain.outputs['misc'])
+            try:
```

Comment:
Not sure if try..except is necessary - `if 'misc' in workchain.outputs` should be fine. 
If no misc output something is very wrong....

Reply:
The reason this is here is that if something is really wrong we would like to have  a clean report where you see the error and not a bunch of errors related to missing `misc` (which is not the true error).